internals/secrethub/service_aws_init.go

```diff
@@ -153,16 +154,18 @@ func (cmd *ServiceAWSInitCommand) Run() error {
 }
 
 // Register registers the command, arguments and flags on the provided Registerer.
-func (cmd *ServiceAWSInitCommand) Register(r command.Registerer) {
+func (cmd *ServiceAWSInitCommand) Register(r cli.Registerer) {
 	clause := r.Command("init", "Create a new service account that is tied to an AWS IAM role.")
-	clause.Arg("repo", "The service account is attached to the repository in this path.").Required().PlaceHolder(repoPathPlaceHolder).SetValue(&cmd.repo)
-	clause.Flag("kms-key", "The ID or ARN of the KMS-key to be used for encrypting the service's account key.").StringVar(&cmd.kmsKeyID)
-	clause.Flag("role", "The role name or ARN of the IAM role that should have access to this service account.").StringVar(&cmd.role)
-	clause.Flag("region", "The AWS region that should be used for KMS.").StringVar(&cmd.region)
-	clause.Flag("description", "A description for the service so others will recognize it. Defaults to `AWS role <role-name>`.").StringVar(&cmd.description)
-	clause.Flag("descr", "").Hidden().StringVar(&cmd.description)
-	clause.Flag("desc", "").Hidden().StringVar(&cmd.description)
-	clause.Flag("permission", "Create an access rule giving the service account permission on a directory. Accepted permissions are `read`, `write` and `admin`. Use `--permission <permission>` to give permission on the root of the repo and `--permission <dir>[/<dir> ...]:<permission>` to give permission on a subdirectory.").StringVar(&cmd.permission)
+	//clause.Arg("repo", "The service account is attached to the repository in this path.").Required().PlaceHolder(repoPathPlaceHolder).SetValue(&cmd.repo)
+	clause.Flags().StringVar(&cmd.kmsKeyID, "kms-key", "", "The ID or ARN of the KMS-key to be used for encrypting the service's account key.")
+	clause.Flags().StringVar(&cmd.role, "role", "", "The role name or ARN of the IAM role that should have access to this service account.")
+	clause.Flags().StringVar(&cmd.region, "region", "", "The AWS region that should be used for KMS.")
+	clause.Flags().StringVar(&cmd.description, "description", "", "A description for the service so others will recognize it. Defaults to `AWS role <role-name>`")
```

Comment:
I identified this default and I was not sure how to handle it.
Should it be put in the normal default version (default "AWS role <role-name>") or keep it in the help text as it is now?
The confusion rose because we have the dynamic part `<role-name>` which is set only after the command is executed. Therefore I was not sure if the default section should contain placeholders as well.

Reply:
Indeed, it does not fill it. 
Nice suggestion. Thanks.

Reply:
Indeed, it doesn't, since this happens at the execution of the command.
The thing is that the default is the string `AWS role <role-name>`, not just the name of the role. Therefore we keep the placeholder as it is.